config.toml

```diff
@@ -13,11 +13,17 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
     weight = 1
     url = "application-faq/"
 
+[[menu.main]]
+    name = "Gold Card Holders profiles"
```

Comment:
An alternative launching point would be "Community", and the eventual page would have the profiles in addition to information about joining the community. What do you think?

Reply:
@fifieldt  Love that idea ❤️

Reply:
My first thought was also around community building. I thought of a list of of "offers and needs". People can list what they can help with and others can list help they need. 

The group here will hear me drill this point in over and and over again. A women identifying community is dear to my heart and would love to see that reflected here. Also, LGTBQ+ and PoC identifying folks would benefits I think for the support of a community. Languages could be useful too.

Of course, the list of possible improvements is endless. The current version is already a great start and certainly directly feel the benefits of such community. 

Thank you all for your time!
Hasnaa